bluebrain/repo-bluebrain/packages/neurodamus-models/package.py

```diff
@@ -72,10 +72,25 @@ def setup_run_environment(self, env):
         env.set(f"NEURODAMUS_{mech.upper()}_ROOT", self.prefix)
         env.set("HOC_LIBRARY_PATH", self.prefix.share.join(f"neurodamus_{mech}").hoc)
         if "+coreneuron" in self.spec:
-            env.set("CORENEURONLIB", self.prefix.lib + "/libcorenrnmech.so")
-        for libnrnmech_name in find(self.prefix.lib, "libnrnmech*.so", recursive=False):
-            env.set("NRNMECH_LIB_PATH", libnrnmech_name)
-            env.set("BLUECELLULAB_MOD_LIBRARY_PATH", libnrnmech_name)
+            import os
```

Comment:
It's a nitpick, but if you still need to change something for the discussion below anyway: the import should probably go at the top of the file. Unless you have a specific reason for putting it here of course!

Reply:
I am afraid flake8 could complain. I will try